aiobotocore/endpoint.py

```diff
@@ -50,8 +55,11 @@ async def convert_to_response_dict(http_response, operation_model):
     elif operation_model.has_event_stream_output:
         response_dict['body'] = http_response.raw
     elif operation_model.has_streaming_output:
-        length = response_dict['headers'].get('content-length')
-        response_dict['body'] = StreamingBody(http_response.raw, length)
+        if httpx and isinstance(http_response.raw, httpx.Response):
```

Comment:
Again, this adds some runtime overhead when httpx is installed, but not used. See my other comment.

Reply:
might be able to get rid of this one with #1365, but I don't see a great way of doing it otherwise.